src/navigation/image_database.cc

```diff
@@ -363,15 +363,15 @@ ImageDatabase::loadCSV()
      * make sense for video-type ones.
      */
     BOB_ASSERT(validIdx(fieldNameIdx[4]) == !isVideoType());
-
+    
     // Assume it's a grid if we have any of the Grid fields...
     if (!hasMetadata()) {
-        m_IsRoute = !std::any_of(fieldNameIdx.cbegin() + 5, fieldNameIdx.cend(), validIdx);
+        m_IsRoute = !std::any_of(fieldNameIdx.cbegin() + 5, fieldNameIdx.cbegin() + 8, validIdx);
```

Comment:
What's this change needed for?

Reply:
It's a bit arbitrary really but pitch and roll data isn't really grid-specific